README.md

```diff
@@ -1,15 +1,18 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+# Tuition Address Book (TAB)
+
+[![CI Status](https://github.com/AY2122S1-CS2103T-F13-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-F13-3/tp/actions)
 [![codecov](https://codecov.io/gh/AY2122S1-CS2103T-F13-3/tp/branch/master/graph/badge.svg?token=ZHYAC5NIS7)](https://codecov.io/gh/AY2122S1-CS2103T-F13-3/tp)
 
+**Tuition Address Book (TAB)** is an all-in-one desktop application for private tutors to manage students and lesson information. TAB comes with a clean Graphical User Interface (GUI) while optimised for user interaction via a CLI (Command Line Interface).
```

Comment:
A suggestion for the **About** description:
>Busy private 1-to-1 home tutors can find it difficult to remember information regarding their many students.
Use **TAB** to manage customer contacts and customer relationships faster than a typical mouse/GUI driven app. **TAB** effortlessly keeps track of large amounts of necessary student and lesson information, to empower tutors to provide the best quality home tuition service.  Use **TAB** to manage customer contacts and customer relationships faster than a typical mouse/GUI driven app!

Reply:
haha sure I'll use this

Reply:
@eeliana @Xiaoyunnn @angkoonhwee any objection?